fh-dataman.json

```diff
@@ -0,0 +1,200 @@
+          "metadata": {
+            "labels": {
+              "name": "fh-dataman"
+            }
+          },
+          "spec": {
+            "containers": [
+              {
+                "name": "fh-dataman",
+                "image": "${FH_DATAMAN_IMAGE}:${FH_DATAMAN_IMAGE_VERSION}",
+                "ports": [
+                  {
+                    "containerPort": "${FH_DATAMAN_PORT}"
+                  }
+                ],
+                "env": [
+                  {
+                    "name": "FHMBAAS_URL",
+                    "value": "${FHMBAAS_URL}"
+                  },
+                  {
+                    "name": "FHMBAAS_KEY",
+                    "value": "${FHMBAAS_KEY}"
+                  },
+                  {
+                    "name": "MONGODB_FHMBAAS_USER",
+                    "valueFrom": {
+                      "configMapKeyRef": {
+                        "name": "mongo-config",
+                        "key": "mongodb-fhmbaas-user"
+                      }
+                    }
+                  },
+                  {
+                    "name": "MONGODB_FHMBAAS_PASSWORD",
+                    "valueFrom": {
+                      "configMapKeyRef": {
+                        "name": "mongo-config",
+                        "key": "mongodb-fhmbaas-password"
+                      }
+                    }
+                  },
+                  {
+                    "name": "FH_LOG_LEVEL",
+                    "value": "${FH_DEFAULT_LOG_LEVEL}"
+                  },
+                  {
+                    "name": "FH_DATAMAN_PORT",
+                    "value": "${FH_DATAMAN_PORT}"
+                  }
+                ],
+                "resources": {
+                  "limits": {
+                    "cpu": "800m",
+                    "memory": "800Mi"
+                  },
+                  "requests": {
+                    "cpu": "200m",
+                    "memory": "200Mi"
+                  }
+                },
+                "imagePullPolicy": "IfNotPresent"
+              }
+            ],
+            "volumes": [
+              {
+                "name": "node-proxy",
+                "configMap": {
+                  "name": "node-proxy"
+                }
+              }
+            ]
+          }
+        }
+      }
+    },
+    {
+      "kind": "Route",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "fh-dataman"
+      },
+      "spec": {
+        "to": {
+          "kind": "Service",
+          "name": "fh-dataman-service"
+        },
+        "tls": {
+          "termination": "edge",
+          "insecureEdgeTerminationPolicy": "Allow"
+        }
+      }
+    }
+  ],
+  "parameters": [
+    {
+    "name": "FHMBAAS_URL",
+    "value": "${FHMBAAS_URL}",
+    "description": "The url to reach fh-mbaas"
+    },
+    {
+    "name": "FHMBAAS_KEY",
+    "value": "${FHMBAAS_KEY}",
+    "description": "The service key of the mbaas"
+    },
```

Comment:
If this needs to be passed in at creation time might be best to move the FHMBAAS_KEY generation to the configmap and read it from there for both here and fh-mbaas since its shared. Otherwise this service needs to be created after. Although we also need the url too and not  sure we can apply that logic there